.vscode/cspell.json

```diff
@@ -39,22 +46,35 @@
     "eventhub",
     "eventhubs",
     "hmac",
+    "incrementalcopy",
```

Comment:
I'd rather not store Storage-specific things in here. Instead, define a `sdk/storage/.dict.txt` file and add an entry in this file like cosmos and identity did. You can put all the Storage-specific things in there you want.

Mainly, the reason is because some of these may very well be misspellings in other contexts. What's here was intended to be more general items. Same with other repos.